_docs/data-model/migrations/migrations.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: defaults
+title: Data Model Migrations
+published: true
+---
+
+### _schemaVersion=0
+
+This migration applies to change introducted as part of the "UTC Bootstrap" ADD_REFERENCE_HERE changes and introduces a generic [`_schemaVersion`](v1#versioning-and-updates) to our data model.
+
+**Check**
+
+In the mongo console check to see if you need to apply the migration by finding how many records don't have the `_schemaVersion`
+
+```
+> db.deviceData.find({"_schemaVersion": {"$exists”:false}}).count()
+```
+
+you should see a count of the records
+
+```
+49903
+```
+
+**Backup**
+
+In mongo backup your existing data collection by performing a copyTo
+
+```
+$ mongoexport --db data --collection deviceData --out deviceData_PreSchemaVersion.json
```

Comment:
Is it the "streams" database?  Or do I have something misconfigured on my machine.  I have the following mongo databases: 

```
> show dbs
data	(empty)
gate	0.203125GB
hydrophone	0.203125GB
local	0.078125GB
messages	0.203125GB
streams	3.9521484375GB
test	(empty)
test_ingestion	0.203125GB
test_streams	0.203125GB
user	0.203125GB
```